frontend/src/components/utils/WalletLink.tsx

```diff
@@ -1,13 +1,19 @@
 import { truncateAccountId } from "../../libraries/formatting";
+import Mixpanel from "../../libraries/mixpanel";
 
 export interface Props {
   accountId: string;
   nearWalletProfilePrefix: string;
 }
 
 const WalletLink = ({ accountId, nearWalletProfilePrefix }: Props) => {
+  const clicked = () =>
+    Mixpanel.track("Click for wallet profile", {
+      accountId: accountId,
+      walletPrefix: nearWalletProfilePrefix,
+    });
   return (
-    <>
+    <div onClick={clicked}>
```

Comment:
I think it should be possible to apply `onClick` directly on `a` tag.

Reply:
If I add `onClick` on a , `target="_blank" rel="noopener"` will be broken